.github/workflows/ci.yml

```diff
@@ -24,6 +24,8 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: yarn install
+      - name: Run lint
+        run: yarn next lint
```

Comment:
Why do we have to call "next lint"? What is next providing here additional to what ESLint already provides? 
If its not much, I'd recommend not adding a dependency on `next` for ESLint.